workflow/setup_expt.py

```diff
@@ -294,7 +65,17 @@ def _update_defaults(dict_in: dict) -> dict:
     # First update config.base
     edit_baseconfig(host, inputs, yaml_dict)
 
-    # loop over other configs and update them
+    # Update stage config
+    stage_dict = {
+        "@ICSDIR@": inputs.icsdir
+    }
+    host_dict = get_template_dict(host.info)
+    stage_dict = dict(stage_dict, **host_dict)
+    stage_input = f'{inputs.configdir}/config.stage_ic'
+    stage_output = f'{inputs.expdir}/{inputs.pslot}/config.stage_ic'
+    edit_config(stage_input, stage_output, stage_dict)
```

Comment:
What is going on here? Why is `config.stage_ic` handled specially?

Reply:
This area of `setup_expt.py` updates templated variables in the configs and places them in the `EXPDIR`. However, other than `config.base`, the other configs handled in this section are just filled in with defaults from a default yaml. I needed to handle `config.stage_ic` like `config.base` and get the user input (or empty default) and host file provided `BASE_IC` into it. There is an entire function just to update `config.base`, that was overkill for `config.stage_ic`, so I added the lines you see above.